zuchini-model/src/main/java/org/zuchini/parser/Visitors.java

```diff
@@ -188,7 +188,8 @@ public Step visitStep(@NotNull GherkinParser.StepContext ctx) {
         @Override
         public Background visitBackground(@NotNull GherkinParser.BackgroundContext ctx) {
             Token keyword = ctx.BACKGROUND_KW().getSymbol();
-            Background background = new Background(feature, keyword.getLine(), keyword.getText(), ctx.lineContent().getText());
+            final String description = ctx.lineContent() == null ? "Background" : ctx.lineContent().getText();
```

Comment:
Do we need a fallback value? Won't null work?

Reply:
I did not want to render `StepContainer#getName` nullable. Nevertheless looking at the Gherkin grammar https://github.com/cucumber/cucumber/blob/master/gherkin/gherkin.berp it seems that *all* descriptions/name (Feature, Scenario, Outline, Background, Examples) are optional. Should zuchini align with Gherkin here? Or should it enforce descriptions on all elements? @jhorstmann

Reply:
I'd actually prefer to enforce describing all those things, but then the error message for missing descriptions needs to be improved.

Reply:
Ok, I propose to also document this "deviation" from the cucumber Gherkin implementation. It's quite surprising to encounter, also given that the file `feature_parser.feature` includes examples without description.